adapters/cisco_nx_rest/do_restore_conf.php

```diff
@@ -47,7 +34,7 @@
 
   sms_set_update_status($sms_csp, $sdid, SMS_OK, $status_type, 'WORKING', "Restoring configuration (restore revision: $revision_id)");
 
-  $ret = me_connect();
+  $ret = cisco_nx_cli_connect();
```

Comment:
I think we need to have a code review with YvesD about the way to handle function/file name in DA.

According to my understanding using ME_... without specifying the device model name in the function/file is better